components/Registry.tsx

```diff
@@ -64,7 +64,7 @@ function Registry(): React.ReactElement {
   }
 
   function getFormattedDateTime(date: Date) {
-    const format = new Intl.DateTimeFormat(undefined, {
+    const format = new Intl.DateTimeFormat([], {
       hour: "numeric",
       minute: "numeric",
       second: "numeric",
```

Comment:
YYYY-MM-DD is non-ambiguous

Reply:
Maybe we just do `2 Nov 2020`? That seems the most obvious.